src/main/java/com/minecolonies/coremod/entity/ai/minimal/EntityAISleep.java

```diff
@@ -160,6 +162,13 @@ public void tick()
         // Go home
         if (!citizen.getCitizenColonyHandler().isAtHome())
         {
+            AbstractJob job = (AbstractJob) citizen.getCitizenData().getJob();
+            if(job.getName() != null) {
+                if (job.getName().contains("Fisherman")) {
+                    EntityAIWorkFisherman fishJob = (EntityAIWorkFisherman) citizen.getCitizenData().getJob().getWorkerAI();
+                    fishJob.retrieveRod();
```

Comment:
probably better to make this an instanceof JobFisherman

Reply:
I couldn't quite get that to work because that was my first thing I tried. Let me go at it again and I'll update it! :D